docs/tutorials/data-loading/local-files.md

```diff
@@ -0,0 +1,131 @@
+# Local files
+
+This tutorial gives examples of how to write scripts to load data from local files such as CSV and Excel files into
+Qlik Associative Engine. It also shows some basic capabilities of load scripts.
+
+!!! Note
+    [halyard.js](https://github.com/qlik-oss/halyard.js) provides functionality to load different kinds of data
+    into Qlik Assocative Engine without having to write load scripts. The tutorial however, focuses on direct use of
+    load scripts in order to explain and show their potential.
+
+## Prerequisites
+
+To follow along in this tutorial, you should have basic understanding of Docker.
+
+You need the following software installed:
+
+* Git
+* Docker
+* docker-compose
+* Node.js
+
+## Setup
+
+To run the example code, clone the [qlik-oss/core-data-loading](https://github.com/qlik-oss/core-data-loading) Git
+repository. Check out the repo documentation to get familiar with the content and structure.
+
+Only one instance of Qlik Associative Engine is needed and it is started with all necessary data files mounted.
+Change `ACCEPT_EULA` as applicable and run:
+
+```sh
+ACCEPT_EULA=<yes/no> docker-compose up -d
+```
+
+Now install dependencies:
+
+```sh
+npm install
+```
+
+## Sample data files
+
+The examples use these files to load data from:
+
+* [airports.csv](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.csv) -
+  Headers on first row, comma-separated data
+* [airports.xlsx](https://github.com/qlik-oss/core-data-loading/blob/master/data/airports.xlsx) -
+  Headers on first row, MS Excel Open XML
```

Comment:
Would it be interesting to show how you would do if the header is on the second row, or something diverging from the first example file?

Reply:
@StefanEnberg and I discussed this and we concluded that we should add such more special cases later if we get feedback on it, and keep it simple for now. You think it is required?